src/components/AppHeader.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <header>
+    <div class="header-wrapper">
+      <div class="container">
+        <nav>
+          <div class="nav-wrapper">
+            <a
+              class="logo-link"
+              href="/"
+            >
+              <img
+                class="logo-image"
+                src="../assets/logo.svg"
+                alt="logo"
+                width="50"
```

Comment:
I think `width` should be in `class="logo-image"`

Reply:
it is in class="logo-image". Using width in html<img> is recommendation